src/wallet/gtest/test_wallet.cpp

```diff
@@ -748,15 +748,9 @@ TEST(WalletTests, GetConflictedSaplingNotes) {
         wtx = wallet.mapWallet[hash];
 
         // Decrypt output note B
-        auto maybe_pt = libzcash::SaplingNotePlaintext::decrypt(
-                consensusParams,
-                wtx.nExpiryHeight,
-                wtx.vShieldedOutput[0].encCiphertext,
-                ivk,
-                wtx.vShieldedOutput[0].ephemeralKey,
-                wtx.vShieldedOutput[0].cmu);
+        auto maybe_pt = wtx.DecryptSaplingNote(Params(), SaplingOutPoint(hash, 0));
         ASSERT_EQ(static_cast<bool>(maybe_pt), true);
```

Comment:
```suggestion
        ASSERT_EQ(maybe_pt.has_value(), true);
```
and similarly below. Non-blocking.